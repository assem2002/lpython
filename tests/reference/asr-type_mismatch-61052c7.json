{
    "basename": "asr-type_mismatch-61052c7",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/type_mismatch.py",
    "infile_hash": "45ee4815e13c9fc4d58572a2f4ec01f5546e1f67878a2913bda89e5c",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-type_mismatch-61052c7.stderr",
    "stderr_hash": "b5474d2ae119f3be899c38b058af70e24aac3cbe38b17e8e7e57e0e7",
    "returncode": 2
}