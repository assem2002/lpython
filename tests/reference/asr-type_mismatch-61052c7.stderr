semantic error: Type mismatch in shorthand operator, the types must be compatible
 --> tests/errors/type_mismatch.py:8:5
  |
8 |     a *= b
  |     ^    ^ type mismatch ('i32' and 'f32')


Note: Please report unclear or confusing messages as bugs at
https://github.com/lcompilers/lpython/issues.
